www/js/sections.js

```diff
@@ -165,6 +177,9 @@ const applyTransformationsToFragment = (fragment, article, isLead) => {
   // 'enwiki > Quadradic equation' and 'enwiki > Away colors > Association football'). See the
   // 'classifyElements' method itself for other examples.
   requirements.themes.classifyElements(fragment)
+
+  lazyImageLoadingTransformer.convertImagesToPlaceholders(fragment)
+  lazyImageLoadingTransformer.loadPlaceholders()
```

Comment:
This should be fine but depending on how your loading works, you may only need to do this once after the page is loaded instead of after every section load.

Reply:
The cool thing about the way we're using fragments is all transforms are complete before a fragment is attached to the `live` document :)

Reply:
Sorry, I was pointing at the call to `loadPlaceholders()` which selectively undoes the transforms.

Reply:
Ah, cool thanks!

Reply:
Thoughts on maybe naming `loadPlaceholders` a bit differently to better distinguish what it does from what `convertImagesToPlaceholders` does?

Reply:
`loadPlaceholders()` transforms placeholders to images. The intent was for the client to think of images as being loaded into the placeholders since the download can fail but naming is tough. Feel free to submit naming improvements in a separate patch that bumps the major version and flag an Android dev.